src/core/server/saved_objects/routes/import.ts

```diff
@@ -64,6 +64,7 @@ export const registerImportRoute = (router: IRouter, config: SavedObjectConfig)
             workspaces: schema.maybe(
               schema.oneOf([schema.string(), schema.arrayOf(schema.string())])
             ),
+            dataSourceEnabled: schema.maybe(schema.boolean({ defaultValue: false })),
```

Comment:
shouldn't this `dataSourceEnabled` option determined by the saved object service itself, passing as parameter doesn't looks right to me

Reply:
Hi, @zengyan-amazon something here might confuse me. The `dataSourceEnabled` is originally from DataSourcePluginSetup props, it is from data source service and pass as service field, UI component will pass it as parameter to import api, UI got it from saved_object_management plugin set up (code)[https://github.com/opensearch-project/OpenSearch-Dashboards/blob/fb76ee9beee3a98d9b7c0783df072abe745acd7a/src/plugins/saved_objects_management/public/plugin.ts#L110], import api from core is used in this plugin, but does not have plugin setup, I do not know how to get the dataSourceEnabled from the saved object client from the core, could you please point me some example

Reply:
Issue to track here: https://github.com/opensearch-project/OpenSearch-Dashboards/issues/6557